proto/lavanet/lava/rewards/query.proto

```diff
@@ -31,6 +32,16 @@ service Query {
   rpc ShowIprpcData(QueryShowIprpcDataRequest) returns (QueryShowIprpcDataResponse) {
     option (google.api.http).get = "/lavanet/lava/rewards/show_iprpc_data";
   }
+
+  // IprpcProviderReward queries for a provider's current IPRPC reward (relative to its serviced CU)
+  rpc IprpcProviderReward(QueryIprpcProviderRewardRequest) returns (QueryIprpcProviderRewardResponse) {
```

Comment:
i think we should make it clear it is only an estimation, lets add it to the names of the querie